assets/src/scripts/_file-click.js

```diff
@@ -39,13 +39,25 @@ const fileClick = async ({ fileName, metadata, url }) => {
   document.querySelector("#openFileName h2").textContent =
     openFile.value.fileName;
 
+  // Create a human readable date from the timestamp
+  const createdAt = new Date(metadata.timestamp).toLocaleDateString(undefined, {
```

Comment:
```suggestion
  const createdAt = new Date(metadata.timestamp).toLocaleDateString("en-GB, {
```

We should default this to "en-GB" so that each OS displays the same setting. Without this, on my Windows install, I actually see a different date style to your screenshot.

Reply:
Ooh, thank you for catching that!